lib/miq_automation_engine/service_models/mixins/miq_ae_service_custom_attribute_mixin.rb

```diff
@@ -2,8 +2,11 @@ module MiqAeServiceCustomAttributeMixin
   extend ActiveSupport::Concern
 
   included do
-    expose :custom_keys, :method => :miq_custom_keys
-    expose :custom_get,  :method => :miq_custom_get
-    expose :custom_set,  :method => :miq_custom_set, :override_return => true
+    expose :migration_status,       :method => :migration_status
```

Comment:
Should not expose any `migration_status` methods in this mixin.

Reply:
Without this change code is not working. Please advice how to make sure that it is if you want to remove it.